models/instrument.py

```diff
@@ -1156,6 +1239,33 @@ def get_short_filter_name(cls, filter):
     # may need specific overrides for some of the steps.  For many
     # instruments, the defaults should work.
 
+    @classmethod
+    def get_filter_bandpasses(cls):
+        """
+        Get a dictionary of filter name -> Bandpass object for a list of common filters.
+        The default Instrument just gives some generic filters and their bandpasses,
+        but subclasses should override (or update) this dictionary with their own
+        filters and bandpasses.
+        """
+
+        # we can probably do better than this, but I don't know if it makes any difference
+        # ref: https://en.wikipedia.org/wiki/Photometric_system
+        values = dict(
```

Comment:
We're probably always going to want to make this instrument-specific.  But, if we're going to have a default, it would probably be better to have something more modern than the Bessel filters here that are usually used with Vega-based magnitudes.  The LSST filters might be a good choice; see section 2.4.1 of:

https://www.lsst.org/sites/default/files/docs/sciencebook/SB_2.pdf